### ISSUE TITLE
[Bug][Expandable] When I Use Expandable In ScriptableWizard

### ISSUE THREAD

**Issue by ZeroUltra** [TylerTemp/SaintsField#1]
```
public class AssetBundleBuildWindow : ScriptableWizard
{
    [SaintsField.Expandable]
    [SerializeField] ABConfig abConfig;
    .....
```
![image](https://github.com/TylerTemp/SaintsField/assets/38253805/d2896a46-f151-49bd-ad27-d20d6f50349e)

**Comment by TylerTemp** [comment]
This foldout actually behaves differently on different unity versions when I test it. Could you give your Unity version?

**Comment by ZeroUltra** [comment]
> This foldout actually behaves differently on different unity versions when I test it. Could you give your Unity version?

Ok!! unity2022.3.16🥰

**Comment by TylerTemp** [comment]
![image](https://github.com/TylerTemp/SaintsField/assets/6391063/b8b3f896-05a3-4ee3-9a46-0ffeb54ebb65)

This should be fixed in [2.0.9](https://github.com/TylerTemp/SaintsField/releases/tag/2.0.9)

Known issue: 
1.  In IMGUI, a custom drawer won't work because `PropertyDrawer` is not allowed to create an `Editor` class, thus it'll list all fields in the object. If the field itself has a custom `PropertyDrawer`, the drawer WILL be used.
2.  In IMGUI, the `Foldout` will NOT be placed at the left space like a Unity's default foldout component, because Unity limited the `PropertyDrawer` to be drawn inside the rect Unity gives. Tryng outside of the rect will make the target non-interactable.
    But in early Unity (like 2019.1), Unity will force `Foldout` to be out of rect on top leve, but not on array/list level... so you may see different outcomes on different Unity version.


Feel free to re-open this issue if it still exists.
